From abebb1bb9b798e50190e208b83cc01babd2089f1 Mon Sep 17 00:00:00 2001
From: MarkusVolk <f_l_k@t-online.de>
Date: Thu, 11 Nov 2021 08:25:33 +0100
Subject: [PATCH] test

Signed-off-by: MarkusVolk <f_l_k@t-online.de>

Hack to make use of tolua from build machine (needs to be installed)
---
diff --git a/meson.build b/meson.build
index 711d639a22..d4e96fa302 100644
--- a/meson.build
+++ b/meson.build
@@ -426,7 +421,7 @@ gitrev = custom_target('gitrev', output: 'fc_gitrev_gen.h',
                        command: ['sh', files('bootstrap/generate_gitrev.sh'), meson.source_root(), '@OUTPUT@'],
                        build_by_default: get_option('gitrev'))
 
-tolua_cmd = find_program('tolua')
+tolua_cmd = find_program('/usr/bin/tolua')
 
 tolua = generator(tolua_cmd,
                   arguments: ['-n', '@BASENAME@',
-- 
2.25.1

